### PR TITLE
fix: change the url path pipelinetemplates to configtemplates

### PR DESCRIPTION
**Pull request by zhujian7**
**What this PR does / why we need it**:

Add your description

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @supereagle @cd1989 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/api/v1/descriptor/template.go [hunk 1]
```diff
@@ -12,17 +12,17 @@ func init() {
...
-		Path:        "/pipelinetemplates",
-		Description: "Template API",
+		Path:        "/configtemplates",
+		Description: "Pipeline Config Template API",
 		Definitions: []definition.Definition{
 			{
 				Method:      definition.List,
 				Function:    handler.ListTemplates,
-				Description: "Get all cyclone built-in pipeline templates",
+				Description: "Get all cyclone built-in pipeline config templates",
```
Comment (supereagle) [inline pkg/api/v1/descriptor/template.go]: Also need to update description and some comments of function `ListTemplates`.

Comment (zhujian7) [reply pkg/api/v1/descriptor/template.go]: have fixed

**Pull request merged**